STM32F4/variants/generic_f407v/ld/common.inc

```diff
@@ -164,11 +150,43 @@ SECTIONS
          * I'm shoving these here naively; there's probably a cleaner way
          * to go about this. [mbolivar]
          */
-        _lm_heap_start = DEFINED(_lm_heap_start) ? _lm_heap_start : _end;
+        _lm_heap_start = DEFINED(_lm_heap_start) ? _lm_heap_start : __ccm_end__;
```

Comment:
ccm_start is not the top address of the CCM, is a label that indicates the end of any variable allocated to CCMRAM (if any) so the heap starts after the last variable there. We can change the name of the label to something else if we want, that name may lead to confusion.
It's in this line:
https://github.com/victorpv/Arduino_STM32/blob/c4233423541f79e7683d2aa43ebebc2cbc356ca1/STM32F4/variants/generic_f407v/ld/common.inc#L186